### PR TITLE
feat:updated reset.css completed Product Listing page

### PR DESCRIPTION
**Pull request by shashikantmaurya0007**
feature: added filter,product list,updated reset.css
Screenshot (1550)
Screenshot (1551)
Screenshot (1552)

### CODE REVIEW

**Review by nyyshaaa — APPROVED** [review]
Looks good to me!

File: ProductListing/productlisting.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+}
+
+.price_range {
+    display: flex;
+    justify-content: space-between;
+}
+
+.filter_container {
+    flex-basis: 35%;
+    box-shadow: rgba(0, 0, 0, 0.25) 0px 54px 55px, rgba(0, 0, 0, 0.12) 0px -12px 30px, rgba(0, 0, 0, 0.12) 0px 4px 6px, rgba(0, 0, 0, 0.17) 0px 12px 13px, rgba(0, 0, 0, 0.09) 0px -3px 5px;
```
Comment (ravikumar0403) [inline ProductListing/productlisting.css:42]: create a variable for this if it's used in multiple places. Changing it later will be easier with a variable

File: ProductListing/productlisting.html [hunk 2]
```diff
@@ -0,0 +1,352 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./productlisting.css" />
+    <title>Document</title>
```
Comment (Sharmaryan) [inline ProductListing/productlisting.html:9]: fix title

Comment (ravikumar0403) [inline ProductListing/productlisting.html:9]: Change the title to be relevant with your page

File: navbar.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    position: sticky;
+    top: 0;
+    width: 100%;
+    z-index: 10;
+    justify-content: space-around;
+    background-color: whitesmoke;
```
Comment (ravikumar0403) [inline navbar.css:12]: use variable for this color as well

**Review by ravikumar0403 — APPROVED** [review]
Left a few comments, Rest all looks good

**Review by Sharmaryan — APPROVED** [review]
everything else is great, suggested a small change

**Pull request merged**